WcaOnRails/app/controllers/persons_controller.rb

```diff
@@ -6,16 +6,14 @@ def index
     respond_to do |format|
       format.html
       format.js do
-        persons = Person.joins("JOIN Countries ON countryId = Countries.id")
+        persons = Person.order(:name, :countryId)
         if params[:region] != "all"
-          persons = persons.where("countryId = :region OR continentId = :region", region: params[:region])
```

Comment:
Fascinatingly enough, @Baiqiang actually pointed out to me earlier today that we have a similar region check in other places in our codebase, and could benefit from conslidation. Mentioning him here so he knows about this other region check.